src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin.cxx

```diff
@@ -397,18 +401,14 @@ add_subset_sensitivity(TargetT& sensitivity, const int subset_num) const
 template<typename TargetT>
 void
 PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin<TargetT>::
-add_view_seg_to_sensitivity(const ViewSegmentNumbers& view_seg_nums) const
+add_view_seg_to_sensitivity(const ViewSegmentTOFNumbers& view_seg_nums) const
```

Comment:
Check the handling of this method, is there a seperate TOF version? How is TOF handled in it?